scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -104,6 +104,7 @@ package object dialects {
     .withAllowParamClauseInterleaving(true).withAllowParameterTypeConversions(true)
 
   implicit val Scala34: Dialect = Scala33.withAllowQuotedTypeVariables(true)
```

Comment:
Since this is available via import, should we not add it to Scala3Future?

Reply:
is the same true for `into` as well?

Reply:
I think so